content/applications/services/helpdesk/advanced/after_sales.rst

```diff
@@ -151,79 +199,100 @@ ticket.
-from the *Helpdesk* ticket, and managed through the *Repairs* application.
+If the ticket is related to an issue with a faulty or broken product, a *repair order* can be
+created from the *Helpdesk* ticket, and managed through the *Repairs* application.
 
 To create a new repair order, open a :menuselection:`Helpdesk` ticket and click on the
-:guilabel:`Repair` button in the upper left corner.
-
-Clicking the :guilabel:`Repair` button opens a blank :guilabel:`Repair Reference` form.
+:guilabel:`Repair` button in the top-left corner. This opens a :guilabel:`Repair Reference` form.
 
 .. image:: after_sales/after-sales-repair-reference.png
    :align: center
    :alt: View of a repair reference page.
 
-.. note::
-   If a product was specified in the :guilabel:`Product` field on the ticket, it will be added to
-   the :guilabel:`Product to Repair` field automatically. If not, click into the field to select a
-   product from the drop down.
+Fill out the fields with the necessary information:
 
-Fill out the :guilabel:`Repair Description` field with a brief explanation of the issue. Click the
-:guilabel:`Sale Order` field and then select the originating :abbr:`SO (Sales Order)` from which the
-product is being repaired from. If a return has been initiated for the product, select the reference
-number from the drop-down in the :guilabel:`Return` field.
+ - :guilabel:`Customer`: this field carries over from the ticket, though a new contact can been
+   selected from the drop-down menu.
+ - :guilabel:`Product to Repair`: if a product was specified in the :guilabel:`Product` field on the
+   ticket, it is added to this field automatically. If not, click into the field to select a product
+   from the drop-down.
+ - :guilabel:`Lot/Serial`: this field is only visible if the products being repaired are tracked via
+   lot or serial numbers.
+ - :guilabel:`Return`: return order from which the product to be repaired comes from.
+ - :guilabel:`Under Warranty`: if this box is checked, the sale price for all products from the
+   repair order are set to zero.
+ - :guilabel:`Scheduled Date`: this field defaults to the current date. To select a new date, click
+   into the field and select a date using the drop-down calendar.
+ - :guilabel:`Responsible`: assign a user from the drop-down menu to manage the repair.
+ - :guilabel:`Tags`: click into this field to assign an existing tag or create a new one. Multiple
+   tags can be assigned.
 
-Choose an :guilabel:`Invoice Method` from the drop-down. Select :guilabel:`Before Repair` or
-:guilabel:`After Repair` to generate an invoice before or after the work is completed. Selecting
-:guilabel:`No Invoice` means that an invoice cannot be generated for this service.
+If parts are required for the repair, they can be added in the :guilabel:`Parts` tab. Additional
+information for the internal repair team can be added to the :guilabel:`Repair Notes` tab.
```

Comment:
As I mentioned previously, I think it would cause more confusion than it would provide actual benefit.  There is just one field, and changing it to a different operation means this is no longer a "repair".